python/lsst/ts/FiberSpectrograph/fiberSpectrograph.py

```diff
@@ -56,20 +89,29 @@ class FiberSpectrograph:
     """`AvsIdentityType` of the connected spectrograph.
     """
 
-    def __init__(self, serial_number=None):
-        self.avaspec = AVS()
-        # NOTE: init(0) initializes the USB library, not device 0.
-        self.avaspec.init(0)
+    def __init__(self, serial_number=None, log=None, log_to_stdout=False):
+        if log is None:
+            self.log = logging.getLogger('FiberSpectrograph')
+        if log_to_stdout:
+            self.log.setLevel(logging.DEBUG)
+            import sys
+            logging.getLogger().addHandler(logging.StreamHandler(sys.stdout))
+
+        self.libavs = ctypes.CDLL("/usr/local/lib/libavs.so.0.2.0")
```

Comment:
`/usr/local/lib` is the installation path of the library from all of the vendor-provided packages I looked at (e.g. Ubuntu, Debian, Centos). The vendor has not provided an updated version since December 2017, and doesn't appear to have done any development since then.

I wish there was a way to tell CDLL to use the LIBRARY_PATH or something, but it only takes a fully-qualified path.

I've converted it to be a global string constant, so at least it's more obvious where to change it.